core/templates/pages/collection-player-page/collection-local-nav/collection-local-nav.directive.html

```diff
@@ -0,0 +1,13 @@
+<ul class="nav navbar-nav oppia-navbar-nav float-right oppia-navbar-pencil-icon">
+  <li ng-if="$ctrl.canEdit" class="nav-item">
+    <a ng-href="/collection_editor/create/<[$ctrl.collectionId]>" uib-tooltip="Edit" tooltip-placement="auto bottom" target="_blank" rel="noopener" class="nav-link">
+      <i class="material-icons">&#xE254;</i>
+      <span class="oppia-icon-accessibility-label">Edit</span>
+    </a>
+  </li>
+</ul>
+<style>
+  collection-local-nav .oppia-navbar-pencil-icon {
+    margin-right: 0;
+  }
+</style>
```

Comment:
Why is this added?

Reply:
Was an accident while resolving merge conflict, Deleting it now